romancal/source_catalog/tests/test_psf.py

```diff
@@ -154,3 +161,25 @@ def test_psf_fit(self, dx, dy, true_flux):
         assert np.all(
             results_table["y_err"] < scale_factor_approx * approx_centroid_err
         )
+
+
+@pytest.mark.stpsf
+def test_create_l3_psf_model():
+    """Test basic results"""
+    psf_model = create_l3_psf_model(filt="F158")
+    assert psf_model.data.shape == (199, 199)
+    assert np.isclose(mad_std(psf_model.data), 0.0)
```

Comment:
It wasn't obvious to me to expect this; is this obvious to you?  It says that the image is mostly zero?

Reply:
This particular one is; it is junk. I can remove that if desired.

Reply:
Oh, sorry, I was referring to fake data, which actually is not an issue in this particular test. I agree that this particular check is probably not good and should be removed.